src/gazebo_gimbal_controller_plugin.cpp

```diff
@@ -483,7 +483,7 @@ void GimbalControllerPlugin::OnUpdate()
     double dt = (time - this->lastUpdateTime).Double();
 
     // We want yaw to control in body frame, not in global.
-    double yaw_command = this->yawCommand + this->lastImuYaw;
+    double yaw_command = this->yawCommand + this->lastImuYaw + M_PI;
```

Comment:
This still doesn't seem well explained.

Reply:
I think the fix should be done by transforming the gimbal orientation in the bodyframe and not by adding the orientation to the command itself. I will try to fix it properly, but this is something that we can hide the issue from the end user

Reply:
Right. So why not use the body frame orientation quaternion directly? You can create a quaternion of the yaw command, apply the rotation and extract yaw.

Reply:
Ah, right. That might be easier. I was trying to keep the yaw command and make everything else in bodyframe